File: examples/linksapp/python-be/app/migrate.py
import asyncio

import forgelib


async def run() -> int:
    reports = await forgelib.ForgeClient.migrate()
    for report in reports:
        print(f"{report.target}: {report.state} ({report.message})")
    return 0 if all(report.state == "applied" for report in reports) else 1


def main() -> None:
    raise SystemExit(asyncio.run(run()))

File: examples/chatapp/node-be/src/migrate.ts
import { ForgeClient } from "forgelib";

const reports = await ForgeClient.migrate();
for (const report of reports) console.error(`${report.target}: ${report.state} (${report.message})`);
if (reports.some((report) => report.state !== "applied")) process.exitCode = 1;
